src/index.html

```diff
@@ -1,13 +1,457 @@
+            </p>
+          </article>
+        </div>
+      </div>
+    </section>
+    <section
+      data-aos="fade-down"
+      class="contacts"
+      id="contactUs"
+    >
+      <div class="container">
+        <h2 class="title contacts__title title--centered">
+          Contact us
+        </h2>
+        <div class="contacts__wrapper">
+          <article class="contacts__museum-info">
+            <h3 class="contacts__museum-name">
+              The Met Fifth Avenue
+            </h3>
+            <p class="contacts__subtitle">
+              Phone
+            </p>
+            <a href="tel:+1 212-535-7710" class="contacts__data">
+              +1 212-535-7710
+            </a>
+            <p class="contacts__subtitle">
+              Address
+            </p>
+            <a
+              href="https://goo.gl/maps/9AzZqeFA5HCNxc7b6"
+              class="contacts__data"
+              target="_blank"
+            >
+              1000 Fifth Avenue
+              <br>
+              New York, NY 10028
+            </a>
+          </article>
+          <article class="contacts__museum-info">
+            <h3 class="contacts__museum-name">
+              The Met Cloisters
+            </h3>
+            <p class="contacts__subtitle">
+              Phone
+            </p>
+            <a href="tel:+1 212-923-3700" class="contacts__data">
+              +1 212-923-3700
+            </a>
+            <p class="contacts__subtitle">
+              Address
+            </p>
+            <a
+              href="https://goo.gl/maps/NQjqGmyp7W72EPes8"
+              class="contacts__data"
+              target="_blank"
+            >
+              99 Margaret Corbin Drive
+              <br>
+              Fort Tryon Park
+              <br>
+              New York, NY 10040
+            </a>
+          </article>
+          <article class="contacts__museum-info">
+            <form action="#" class="form">
+              <label>
+                <input
+                  class="form__input"
+                  type="text"
+                  placeholder="Name"
+                  name="name"
+                  required
+                >
+              </label>
+              <label>
+                <input
+                  class="form__input"
+                  type="email"
+                  placeholder="Email"
+                  name="email"
+                  required
+                >
+              </label>
+              <textarea
+                name="message"
+                placeholder="Message"
+                class="form__input form__input--message"
+                required
+              ></textarea>
+              <button type="submit" class="action-button">
+                Send
+              </button>
+            </form>
+          </article>
+        </div>
+      </div>
+    </section>
+    <footer class="footer"></footer>
+    <script src="./scripts/main.js"></script>
```

Comment:
Remove the second script tag for main.js (line 455) since the main.js file is loaded twice in your code.